cmd/homi/docker/service/validator.go

```diff
@@ -159,6 +159,11 @@ var validatorTemplate = `{{ .Name }}:
 {{- end }}
         echo 'ADDITIONAL="$$ADDITIONAL --debug --metrics --prometheus"' >> /klaytn-docker-pkg/conf/k{{ .NodeType }}d.conf
         /klaytn-docker-pkg/bin/k{{ .NodeType }}d start
+{{- if eq .NodeType "cn" "scn"}}
```

Comment:
@kjhman21 I am sorry. I tested this and SCN can not get `.Address`. I will fix it later. T.T
```suggestion
{{- if eq .NodeType "cn"}}
```